src/components/content/monitor/Monitor.tsx

```diff
@@ -0,0 +1,326 @@
+                Tip('error', 'fetch all deployed services error.');
+                setQueryResult(true);
+                console.error('fetch all deployed services error: ', error);
+            });
+    }, []);
+
+    const handleChangeServiceName = (selectServiceName: string) => {
+        if (selectServiceName) {
+            const cusServiceNameList: { value: string; label: string }[] = [];
+            deployedServiceList.forEach((serviceVo: ServiceVo) => {
+                if (serviceVo.name === selectServiceName) {
+                    const cusServiceName: { value: string; label: string } = {
+                        value: serviceVo.customerServiceName ?? '',
+                        label: serviceVo.customerServiceName ?? '',
+                    };
+                    cusServiceNameList.push(cusServiceName);
+                }
+            });
+            setCustomerServiceNameList(cusServiceNameList);
+        }
+    };
+
+    const handleChangeCustomerServiceName = (selectCustomerServiceName: string) => {
+        form.setFieldsValue({ ServiceId: getCurrentServiceId(selectCustomerServiceName) });
+    };
+
+    const getCurrentServiceId = (selectCustomerServiceName: string): string => {
+        let currentServiceID: string = '';
+        if (selectCustomerServiceName) {
+            deployedServiceList.forEach((serviceVo) => {
+                if (selectCustomerServiceName === serviceVo.customerServiceName) {
+                    currentServiceID = serviceVo.id;
+                }
+            });
+        }
+        return currentServiceID;
+    };
+
+    const layout = {
+        labelCol: { span: 8 },
+        wrapperCol: { span: 16 },
+    };
+
+    const tailLayout = {
+        wrapperCol: { offset: 8, span: 16 },
+    };
+
+    const onFinish = (values: { ServiceName: string; CustomerServiceName: string; ServiceId: string }) => {
+        monitorMetricsQueue.clear();
+        if (fetchMonitorMetricDataTimer) {
+            clearInterval(fetchMonitorMetricDataTimer);
+        }
+        if (!values.ServiceId) {
+            return;
+        }
+        const selectedServiceId = values.ServiceId;
+        setServiceId(selectedServiceId);
+        void MonitorService.getMetrics(selectedServiceId)
+            .then((rsp: Metric[]) => {
+                if (rsp.length > 0) {
+                    monitorMetricsQueue.enqueue(rsp);
+                    showMonitorMetrics(monitorMetricsQueue);
+                    startFetchMonitorMetricDataTimer(monitorMetricsQueue, selectedServiceId);
+                }
+            })
+            .catch((error: Error) => {
+                Tip('error', 'Get metrics of the deployed service error.');
+                setQueryResult(true);
+                console.error('Get metrics of the deployed service error.', error);
+            });
+    };
+
+    const startFetchMonitorMetricDataTimer = (monitorMetricsQueue: Queue<Metric[]>, selectedServiceId: string) => {
```

Comment:
is it possible to provide a option in GUI to disable auto refresh of metrics data? Would be good to have it.